examples/cdn/frontdoor/frontdoor-managed-ssl-certificate-with-multiple-custom-domains/main.tf

```diff
@@ -0,0 +1,319 @@
+  cdn_frontdoor_endpoint_id     = azurerm_cdn_frontdoor_endpoint.example.id
+  cdn_frontdoor_origin_group_id = azurerm_cdn_frontdoor_origin_group.example.id
+  cdn_frontdoor_origin_ids      = [azurerm_cdn_frontdoor_origin.example.id]
+  enabled                       = true
+
+  forwarding_protocol        = "HttpsOnly"
+  https_redirect_enabled     = true
+  patterns_to_match          = ["/*"]
+  supported_protocols        = ["Http", "Https"]
+  cdn_frontdoor_rule_set_ids = [azurerm_cdn_frontdoor_rule_set.example.id]
+
+  cache {
+    compression_enabled           = true
+    content_types_to_compress     = ["text/html", "text/javascript", "text/xml"]
+    query_strings                 = ["account", "settings"]
+    query_string_caching_behavior = "IgnoreSpecifiedQueryStrings"
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "contoso" {
+  name                     = "${var.prefix}-contoso-custom-domain"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+  dns_zone_id              = azurerm_dns_zone.example.id
+  host_name                = join(".", ["contoso", azurerm_dns_zone.example.name])
+
+  tls {
+    certificate_type    = "ManagedCertificate"
+    minimum_tls_version = "TLS12"
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "fabrikam" {
+  name                     = "${var.prefix}-fabrikam-custom-domain"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+  dns_zone_id              = azurerm_dns_zone.example.id
+  host_name                = join(".", ["fabrikam", azurerm_dns_zone.example.name])
+
+  tls {
+    certificate_type    = "ManagedCertificate"
+    minimum_tls_version = "TLS12"
+  }
+}
+
+resource "azurerm_dns_txt_record" "contoso" {
+  name                = join(".", ["_dnsauth", "contoso"])
+  zone_name           = azurerm_dns_zone.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  ttl                 = 3600
+
+  record {
+    value = azurerm_cdn_frontdoor_custom_domain.contoso.validation_properties.0.validation_token
+  }
+}
+
+resource "azurerm_dns_txt_record" "fabrikam" {
+  name                = join(".", ["_dnsauth", "fabrikam"])
+  zone_name           = azurerm_dns_zone.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  ttl                 = 3600
+
+  record {
+    value = azurerm_cdn_frontdoor_custom_domain.fabrikam.validation_properties.0.validation_token
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain_txt_validator" "contoso" {
+  cdn_frontdoor_custom_domain_id = azurerm_cdn_frontdoor_custom_domain.contoso.id
+  dns_txt_record_id              = azurerm_dns_txt_record.contoso.id
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain_txt_validator" "fabrikam" {
+  cdn_frontdoor_custom_domain_id = azurerm_cdn_frontdoor_custom_domain.fabrikam.id
+  dns_txt_record_id              = azurerm_dns_txt_record.fabrikam.id
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain_route_association" "example" {
```

Comment:
This resource type has been removed from the provider.  I think this example needs to be updated.

Reply:
@ossentoo, yep... as I was still moving the schema around a bit I didn't want to change the example files until I was fairly certain that the schema wasn't going to move too much anymore. I will be fixing that and adding a CMK/BYOK example in there as well a bit later tonight. 🚀